megamek/src/megamek/client/bot/princess/PathRanker.java

```diff
@@ -53,28 +52,14 @@ public PathRanker(Princess princess) {
         owner = princess;
     }
 
-    /**
-     * Gives the "utility" of a path; a number representing how good it is.
-     * Rankers that extend this class should override this function
-     */
-    RankedPath rankPath(MovePath path, Game game) {
```

Comment:
Let's not remove this, in case we want to override the PathRanker class in the future without overriding the rankPath method.

Reply:
It's completely unused, so there's no reason to keep it. It'll still be in our history if that ends up being needed at some point in the future.